scripts/openroad/ioplacer.tcl

```diff
@@ -11,35 +11,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+source $::env(SCRIPTS_DIR)/openroad/common/io.tcl
+read
 
 if { [info exists ::env(CONTEXTUAL_IO_FLAG)] } {
 	read_lef $::env(placement_tmpfiles)/top_level.lef
-	#ppl::set_num_slots 2
-}
-
-if {[catch {read_lef $::env(MERGED_LEF)} errmsg]} {
-	puts stderr $errmsg
-	exit 1
-}
-
-if {[catch {read_def $::env(CURRENT_DEF)} errmsg]} {
-	puts stderr $errmsg
-	exit 1
 }
 
 if {$::env(FP_IO_HLENGTH) != "" && $::env(FP_IO_HLENGTH) != ""} {
```

Comment:
not part of this pr but ummm

Reply:
I meant to comment on this `if {$::env(FP_IO_HLENGTH) != "" && $::env(FP_IO_HLENGTH) != ""} {`

Reply:
You know whose avatar this is. :) 
![image](https://user-images.githubusercontent.com/12652988/189652566-719cd376-f2cc-4117-976d-55b20b6bb3d1.png)